.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = { 
+  extends: "airbnb-base",
+  parserOptions: {
+    ecmaFeatures: {
+      'jsx': true
```

Comment:
Why airbnb-base plus an option to enable JSX, rather than just using the normal airbnb style guide, which handles React by default?

Reply:
Fixed!

Reply:
Actually, I'm going to stick to airbnb-base eslint for now...I need to re-configure all eslint